### PR TITLE
add note callout regarding multiple OIDC providers

### LINKED ISSUE #2644: Call out known caveat regarding configuration of multiple OIDC providers
### Description

A backend with more than one OIDC provider configured will behave in an undefined way.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages

https://docs.sensu.io/sensu-go/latest/operations/control-access/auth/#openid-connect-10-protocol-oidc-authentication

### Expected behavior

The docs warn about known caveats,

### Corrections needed

Add a warning or similar callout indicating that configuring more than one OIDC provider can lead to undefined behavior. For proper operation of OIDC authentication, only one OIDC provider should be configured.

### Additional context

https://github.com/sensu/sensu-enterprise-go/issues/1175

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by cwjohnston**
## Description

Add a callout to document known caveat when multiple OIDC providers are defined.

## Motivation and Context

Close #2644 

## Review Instructions

I'll copy the changes to other versions once the content is dialed in.

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@cwjohnston I think this addresses the same issue as https://github.com/sensu/sensu-docs/pull/2645, is that right? It sounds to me like this closes https://github.com/sensu/sensu-docs/pull/2644, so I think they are the same. I was going to close https://github.com/sensu/sensu-docs/pull/2645 in favor of this PR but thought I'd better be sure first.

edit: never mind, I am sure of it now that I reviewed. I'm going to close https://github.com/sensu/sensu-docs/pull/2645. I think this one closes https://github.com/sensu/sensu-docs/issues/2644

File: content/sensu-go/5.18/operations/control-access/auth.md [hunk 1]
```diff
@@ -1092,6 +1092,10 @@ Sensu offers [commercial support][6] for the OIDC provider for using the OpenID
 
 The Sensu OIDC provider is tested with [Okta][51] and [PingFederate][52].
 
+{{% notice note %}}
+**NOTE**: Defining multiple OIDC providers defined can lead to inconsistent authentication behavior. Use `sensuctl auth list` to verify that only one authentication provider of type `OIDC` is defined. Use `sensuctl auth delete $NAME` to remove excess OIDC provider configurations by name.
```
Comment (hillaryfraley) [inline content/sensu-go/5.18/operations/control-access/auth.md]:
```suggestion
**NOTE**: Defining multiple OIDC providers can lead to inconsistent authentication behavior.
Use `sensuctl auth list` to verify that only one authentication provider of type `OIDC` is defined.
If more than one OIDC auth provider configuration is listed, use `sensuctl auth delete $NAME` to remove the extra OIDC configurations by name.
```

**Comment by cwjohnston** [comment]
~Updated issue description to link correct issue. Sorry for the confusion. I think I can also update this to close #2648~

**Review by hillaryfraley — APPROVED** [review]
👍

**Pull request merged**